.gofmt.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+fmt_errors=$(gofmt -d .)
```

Comment:
any particular reason we chose to use `gofmt -l -s`? that will only list the affected files, not the actual errors contained within them. i can see arguments for both ways (error verbosity in travis runs may or may not be a good thing), but that's why i chose to use `gofmt -d`.

totally agree we should be consist, just wanna make sure we agree on what we're consist on :)